pulpanalytics/tests/test_summarize.py

```diff
@@ -67,3 +69,26 @@ def test_summary_worker_count(monkeypatch, db, persistent_min_age_days):
         assert daily_summary.summary.online_workers == Summary.OnlineWorkers(
             processes__avg=2, hosts__avg=1
         )
+
+
+def test_summary_postgresql_version(monkeypatch, db):
```

Comment:
@bmbouter can you follow up with a version that takes `persistent_min_age_days` as a fixture?
(Either you create fake systems that report twice, or you assert that nothing will be collected...).